Gruntfile.js

```diff
@@ -135,6 +135,8 @@ module.exports = function (grunt) {
                             '!thirdparty/CodeMirror2/mode/**/*.html',
                             '!thirdparty/CodeMirror2/**/*test.js',
                             'thirdparty/CodeMirror2/theme/{,*/}*',
+                            'thirdparty/filer-dialogs/bower_components/font-awesome/css/font-awesome.min.css ',
```

Comment:
I'd still like to find a way to not incur the cost of downloading these fonts + font-awesome, especially for cases where the dialogs are never shown.  Could we somehow load that CSS *only* the first time we show those dialogs instead of always having to load it?  It wouldn't be a huge deal to download those fonts after the app has started.